env_config/env.docker

```diff
@@ -89,3 +89,10 @@ N8N_BASIC_AUTH_ACTIVE=true
 N8N_BASIC_AUTH_USER=n8n
 N8N_BASIC_AUTH_PASSWORD=n8n
 N8N_ENCRYPTION_KEY=secret
+
+# ------------------------------------------------------------------------------
+# Clamav - Virus scanning
+# ------------------------------------------------------------------------------
+CLAMAV_PORT=3310
+CLAMAV_HOST=clamav
+SHOULD_CLAMAV_SCAN=true
```

Comment:
Wondering if there is a better name for this.

Like, `ENABLE_CLAMAV` or something

Reply:
Or maybe something more descriptive, like `ENABLE_FILE_VIRUS_SCAN`